src/modules/nodejs.rs

```diff
@@ -71,6 +71,20 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
                 }
                 _ => None,
             })
+            .map(|variable| match variable {
+                "expected_version" => {
+                    let version = nodejs_version.as_deref();
+                    let expected_version = get_engines_version(context);
+                    let versions_in_range =
+                        check_engines_version(version, get_engines_version(context));
+
+                    match (expected_version, versions_in_range) {
+                        (Some(ver), false) => Some(Ok(ver)),
+                        _ => None,
+                    }
+                }
+                _ => None,
+            })
```

Comment:
Please re-use the `map` handling `$version`

Reply:
So instead of a separate variable, it'll show the expected version if the installed one is wrong? Like it'll replace the normal version variable?

Reply:
Please just move the `match`-arm for `$expected_version` into the existing `match` used by `$version`.

Reply:
Oh! Okay sure thing